### PR TITLE
MediaFormat implements Parcelable

### PR DESCRIPTION
**Pull request by b95505017**
So that we could export ExoPlayer track format to another process

### CODE REVIEW

File: library/src/main/java/com/google/android/exoplayer/MediaFormat.java [hunk 1]
```diff
@@ -15,22 +15,25 @@
  */
 package com.google.android.exoplayer;
 
+import android.annotation.SuppressLint;
```
Comment (ojw28) [inline library/src/main/java/com/google/android/exoplayer/MediaFormat.java]: Please revert the change of import order (so android ones are still below the com.google ones).

File: library/src/main/java/com/google/android/exoplayer/MediaFormat.java [hunk 2]
```diff
@@ -214,6 +217,29 @@ public static MediaFormat createId3Format() {
         C.UNKNOWN_TIME_US);
   }
 
+  /* package */ MediaFormat(Parcel in) {
```
Comment (ojw28) [inline library/src/main/java/com/google/android/exoplayer/MediaFormat.java]: Would be nice if the ordering here is the same as in the constructor below, rather than being slightly different ;).

Comment (b95505017) [inline library/src/main/java/com/google/android/exoplayer/MediaFormat.java]: All done! :)
